ansible/roles/test/tasks/fdb_mac_expire.yml

```diff
@@ -0,0 +1,109 @@
+- fail: msg="testbed_type is not defined"
+  when: testbed_type is not defined
+
+- fail: msg="testbed_type {{test_type}} is invalid"
+  when: testbed_type not in ['t0', 't0-64', 't0-64-32', 't0-116']
+
+- include_vars: "vars/topo_{{testbed_type}}.yml"
+
+- name: Expand properties into props
+  set_fact: props="{{configuration_properties['common']}}"
+
+- name: Gather minigraph facts about the device
+  minigraph_facts: host={{inventory_hostname}}
+
+- name: Copy tests to PTF
+  copy: src=roles/test/files/ptftests dest=/root
+  delegate_to: "{{ptf_host}}"
+
+- name: Copy FDB information file to PTF
+  template: src=roles/test/templates/fdb.j2 dest=/root/fdb_info.txt
+  delegate_to: "{{ ptf_host }}"
+
+- name: clear fdb all
+  shell: clear fdb all
+
+- name: copy current switch.json from docker to host
+  shell: docker cp swss:/etc/swss/config.d/switch.json .
+
+- name: set fdb value to "{{fdb_aging_time}}"
+  replace:
+    dest: switch.json
+    regexp: '"fdb_aging_time": ".*"'
+    replace: '"fdb_aging_time": "{{fdb_aging_time}}"'
+  become: true
+
+- name: copy current switch.json from host to docker
+  shell: docker cp switch.json swss:/etc/swss/config.d/switch.json
+
+- name: run swssconfig switch.json command in container swss
+  shell: docker exec -it swss bash -c "swssconfig /etc/swss/config.d/switch.json"
```

Comment:
Not sure how it passed for you with the` "-it"` option, if I run, I get "the input device is not a TTY". You would need to remove this option.

Reply:
I passes in LKDN environment with -it. I will change it without -it and see if it passes, then I will remove this option for github as well. Thx